frontend/src/main/pages/Reviews/PostReviewPage.js

```diff
@@ -1,14 +1,56 @@
 import BasicLayout from "main/layouts/BasicLayout/BasicLayout";
-import { useParams } from "react-router-dom";
+import { useNavigate, useParams } from "react-router-dom";
+import React from "react";
+import axios from "axios";
+import { toast } from "react-toastify";
+import ReviewForm from "main/components/MyReviews/ReviewForm";
+import { useBackend } from "main/utils/useBackend";
 
 export default function PostReviewPage() {
+  const navigate = useNavigate();
   const { id } = useParams();
 
+  const { data: menuItems } = useBackend(
+    // Stryker disable next-line all : don't test internal caching of React Query
+    [`/api/diningcommons/menuitem?id=${id}`],
+    // Stryker disable next-line all
+    {
+      // Stryker disable next-line all : the default method is get, so replacing with an empty string will do nothing
+      method: "GET",
+      // Stryker disable next-line all : id will be defined
+      url: `/api/diningcommons/menuitem?id=${id}`,
+    },
+    // Stryker disable next-line all : Don't test empty initial data
+    [],
+  );
+
+  const itemName = menuItems.name;
+
+  const submitReview = async (formData) => {
+    try {
+      await axios.post("/api/reviews/post", null, {
+        params: {
+          itemId: id,
+          reviewerComments: formData.reviewerComments,
+          itemsStars: formData.itemsStars,
+          dateItemServed: formData.dateItemServed,
+        },
+      });
+      // Stryker disable next-line all : itemName will be defined
+      toast(`Review submitted for ${itemName}`);
+      navigate("/myreviews");
```

Comment:
If you remove the `/myreviews` page, which I suggest, since it currently serves *absolutely no purpose whatsoever*, I suggest you replace this with `navigate(-1)` so that the app returns to the page that it just came from.  This will be a much better user experience.